frontend/src/pages/Login/Login.jsx

```diff
@@ -7,10 +7,13 @@ const uiConfig = {
   signInOptions: [firebase.auth.GoogleAuthProvider.PROVIDER_ID],
   callbacks: {
     signInSuccessWithAuthResult: (authResult, redirectUrl) => {
-      console.log(authResult); // eslint-disable-line no-console
+      console.log(authResult.credential.idToken); // eslint-disable-line no-console
+      document.cookie = `threaderAuthToken=${JSON.stringify(
+        firebase.auth().currentUser.getIdToken(false),
+      )}`;
       console.log(redirectUrl); // eslint-disable-line no-console
       console.log(firebase.auth().currentUser.getIdToken(false)); // eslint-disable-line no-console
-      return false;
+      return true;
```

Comment:
Does this work? Documentation suggests that the 'signInSuccessUrl' is 'Required when the signInSuccessWithAuthResult callback is not used or when it returns true.'

Reply:
It doesn't matter whether or not its true or false.